rotkehlchen/globaldb/updates.py

```diff
@@ -443,7 +443,8 @@ def _handle_asset_update(
         """
         local_asset: Optional[Asset] = None
         with suppress(UnknownAsset):
-            local_asset = Asset(remote_asset_data.identifier).check_existence()
+            # we avoid querying the packaged db to prevent the copy of constant assets
+            local_asset = Asset(remote_asset_data.identifier).check_existence(query_packaged_db=False)  # noqa: E501
```

Comment:
1. This extra condition here prevents us from introducing this issue without noticing in the future. It might happen that we forget about this constraint and the bug appears again with us not noticing.  https://discord.com/channels/657906918408585217/683245846070165504/1066110368402899065


For the bug you have to keep in mind that we have 3 globaldbs:
1. The user globaldb
2. The packaged globaldb
3. An intermediate globaldb that is a copy of 1. and where we apply the updats

The sequence for the infinite loop was:

1. Update starts
2. OP_ETH appeas in the update and by checking its existence we copy it since is in the constants assets we copy it
3. When trying to apply again the update with the conflicts solved a new one appears, `OP_ETH` and is the only conflict now
4. The state of the conflicts is not kept and since you have only resolved the conflict with `OP_ETH` the others are still there and this restarts the chain

Reply:
Ah yes, I see, so it gets added to the real globaldb, not the copy that is used for updating. Yeah, alright, thanks for the explanation :+1: